main.go

```diff
@@ -37,6 +37,7 @@ var (
 	flagInsecureAllowHTTP  bool
 	flagCompression        string
 	flagVersion            bool
+	flagOCISupport         bool
```

Comment:
Whoops, forgot to remove this. This is leftover from when the PR was to add a flag to disable OCI support. Removing now.